3_Lesson/lesson3-1.py

```diff
@@ -0,0 +1,23 @@
+"""
+ Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+ Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+ В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+
+*Пример:*
+
+5
+    1 2 3 4 5
+    3
+    -> 1
+"""
+
+n = int(input())
+list_1=[]
+for i in range(n):
+    list_1.append(int(input()))
+m=int(input())
+k=0
+for i in range(n):
+    if list_1[i]==m:
```

Comment:
Можно проходиться напрямую по элементам списка
```suggestion
for i in list_1:
    if i ==m:
```